references/dbt-projects.mdx

```diff
@@ -170,9 +170,9 @@ my-snowflake-db:
       type: snowflake
       account: [account id]
 
-      # User/password auth
-      user: [username]
-      password: [password]
+      # User/private_key auth
+      private_key_path: [path/to/private.key]
```

Comment:
There are 2 ways of adding private keys , 
- one is with `private_key_path` pointing to the path of the private (p8) key 
- the other is `private_key` with the `content` of the p8 key (encrypted or unencrypted) 

Might be worth mentioning this somewhere.